fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraRepositoryImpl.java

```diff
@@ -220,6 +221,33 @@ public FedoraObject createObject(final String path) throws FedoraException {
     }
 
     @Override
+    public FedoraObject createObject() throws FedoraException {
```

Comment:
Why is the `createObject()` method on both the FedoraRepositoryImpl and FedoraObjectImpl classes?